ush/config_defaults.yaml

```diff
@@ -1177,6 +1183,15 @@ nco:
 # MAKE GRID config parameters
 #-----------------------------
 task_make_grid:
+  execution:
+    nodes: 1
+    cores: 24
+    walltime: 00:20:00
+  # Variables used in run script
+  envars:
+    GRID_DIR: '{{ task_make_grid.GRID_DIR }}'
+    GFDLgrid_NUM_CELLS: '{{ task_make_grid.GFDLgrid_NUM_CELLS }}'
+    GFDLgrid_USE_NUM_CELLS_IN_FILENAMES: '{{ task_make_grid.GFDLgrid_USE_NUM_CELLS_IN_FILENAMES }}'
```

Comment:
There are a LOT of variables in there that aren't used in the ex-script, so I cheated a little on this one and only referenced a small subset of what's in the larger section.

This will likely change significantly with @elcarpenterNOAA's make_grid integration work.